app-text/pandoc/pandoc-2.14.1.ebuild

```diff
@@ -0,0 +1,103 @@
+	>=dev-haskell/http-types-0.8:=[profile?] <dev-haskell/http-types-0.13:=[profile?]
+	>=dev-haskell/ipynb-0.1:=[profile?] <dev-haskell/ipynb-0.2:=[profile?]
+	>=dev-haskell/jira-wiki-markup-1.4:=[profile?] <dev-haskell/jira-wiki-markup-1.5:=[profile?]
+	>=dev-haskell/juicypixels-3.1.6.1:=[profile?] <dev-haskell/juicypixels-3.4:=[profile?]
+	>=dev-haskell/mtl-2.2:=[profile?] <dev-haskell/mtl-2.3:=[profile?]
+	>=dev-haskell/network-2.6:=[profile?]
+	>=dev-haskell/network-uri-2.6:=[profile?] <dev-haskell/network-uri-2.8:=[profile?]
+	>=dev-haskell/pandoc-types-1.22:=[profile?] <dev-haskell/pandoc-types-1.23:=[profile?]
+	>=dev-haskell/parsec-3.1:=[profile?] <dev-haskell/parsec-3.2:=[profile?]
+	>=dev-haskell/random-1:=[profile?] <dev-haskell/random-1.3:=[profile?]
+	>=dev-haskell/safe-0.3.18:=[profile?] <dev-haskell/safe-0.4:=[profile?]
+	>=dev-haskell/scientific-0.3:=[profile?] <dev-haskell/scientific-0.4:=[profile?]
+	>=dev-haskell/sha-1.6:=[profile?] <dev-haskell/sha-1.7:=[profile?]
+	>=dev-haskell/skylighting-0.11:=[profile?] <dev-haskell/skylighting-0.12:=[profile?]
+	>=dev-haskell/skylighting-core-0.11:=[profile?] <dev-haskell/skylighting-core-0.12:=[profile?]
+	>=dev-haskell/split-0.2:=[profile?] <dev-haskell/split-0.3:=[profile?]
+	>=dev-haskell/syb-0.1:=[profile?] <dev-haskell/syb-0.8:=[profile?]
+	>=dev-haskell/tagsoup-0.14.6:=[profile?] <dev-haskell/tagsoup-0.15:=[profile?]
+	>=dev-haskell/temporary-1.1:=[profile?] <dev-haskell/temporary-1.4:=[profile?]
+	>=dev-haskell/texmath-0.12.3:=[profile?] <dev-haskell/texmath-0.12.4:=[profile?]
+	>=dev-haskell/text-1.1.1.0:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-haskell/text-conversions-0.3:=[profile?] <dev-haskell/text-conversions-0.4:=[profile?]
+	>=dev-haskell/unicode-collation-0.1.1:=[profile?] <dev-haskell/unicode-collation-0.2:=[profile?]
+	>=dev-haskell/unicode-transforms-0.3:=[profile?] <dev-haskell/unicode-transforms-0.4:=[profile?]
+	>=dev-haskell/unordered-containers-0.2:=[profile?] <dev-haskell/unordered-containers-0.3:=[profile?]
+	>=dev-haskell/xml-1.3.12:=[profile?] <dev-haskell/xml-1.4:=[profile?]
+	>=dev-haskell/xml-conduit-1.9.1.1:=[profile?] <dev-haskell/xml-conduit-1.10:=[profile?]
+	>=dev-haskell/zip-archive-0.2.3.4:=[profile?] <dev-haskell/zip-archive-0.5:=[profile?]
+	>=dev-haskell/zlib-0.5:=[profile?] <dev-haskell/zlib-0.7:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+	trypandoc? ( >=dev-haskell/wai-0.3:=[profile?]
+			>=dev-haskell/wai-extra-3.0.24:=[profile?] )
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.2.0.1
+	test? ( >=dev-haskell/diff-0.2 <dev-haskell/diff-0.5
+		>=dev-haskell/quickcheck-2.4 <dev-haskell/quickcheck-2.15
+		>=dev-haskell/tasty-0.11 <dev-haskell/tasty-1.5
+		>=dev-haskell/tasty-golden-2.3 <dev-haskell/tasty-golden-2.4
+		>=dev-haskell/tasty-hunit-0.9 <dev-haskell/tasty-hunit-0.11
+		>=dev-haskell/tasty-lua-0.2 <dev-haskell/tasty-lua-0.3
+		>=dev-haskell/tasty-quickcheck-0.8 <dev-haskell/tasty-quickcheck-0.11 )
+"
+
```

Comment:
Previous ebuild turns off optimisations by default due to reportedly high memory usage starting with the ghc-8.8 series. For me it seems okay now on ghc-8.10.5, is it also for you?

Reply:
The memory seems to be manageable for me as well.